engine-src/Game/LambdaHack/Client/UI/Content/Input.hs

```diff
@@ -263,7 +252,7 @@ dropItems t = ([CmdItemMenu, CmdItem], t, dropCmd)
 
 descIs :: [TriggerItem] -> Text
 descIs [] = "trigger an item"
-descIs (t : _) = makePhrase [tiverb t, tiobject t]
+descIs _ = makePhrase ["fling", "in-range projectile"]
```

Comment:
I'd rather inline the whole `makePhrase ["fling", "in-range projectile"]` into `projectI`, because `descIs` is also used in `applyIK` so we can't change it freely.